frontend/cypress/stubs/orgSignUp/mappings/api_v1_users-d2c41957-0331-4c20-a597-7002630980ab.json

```diff
@@ -5,14 +5,14 @@
     "url" : "/api/v1/users?activate=false",
     "method" : "POST",
     "bodyPatterns" : [ {
-      "equalToJson" : "{\"profile\":{\"firstName\":\"Greg\",\"lastName\":\"McTester\",\"login\":\"greg@mailinator.com\",\"email\":\"greg@mailinator.com\"},\"groupIds\":[\"00g228nyd7uuGaG1L1d7\",\"00g228nrj9QDJ10y61d7\"]}",
+      "equalToJson" : "{\"profile\":{\"firstName\":\"Greg\",\"lastName\":\"McTester\",\"login\":\"${json-unit.any-string}\",\"email\":\"${json-unit.any-string}\"},\"groupIds\":[\"00g228nyd7uuGaG1L1d7\",\"00g228nrj9QDJ10y61d7\"]}",
```

Comment:
Consider checking for email-like strings here (contain `@` and `.`)

Reply:
I am having the darndest time getting regex to work on this Wiremock field. The more I struggle to get it working, the more I think about how Faker is filling in this data and our app is validating the input. I think we'll be ok with the other hard-coded values and accept whatever is here. Thoughts?